x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/host_isolation_exceptions.ts

```diff
@@ -187,8 +198,8 @@ export default function ({ getService }: FtrProviderContext) {
             .set('kbn-xsrf', 'true')
             .send(body)
             .expect(400)
-            .expect(anEndpointArtifactError)
-            .expect(anErrorMessageWith(/\[osTypes\]: array size is \[2\]/));
+            .expect(anEndpointArtifactError);
+          expect(anErrorMessageWith(/\[osTypes\]: array size is \[3\]/));
```

Comment:
Check the error returned. I'm not sure this is correct - but lets see if the test passes

Reply:
FYI: I was thinking that `array size is [2]` would be returned, along with some other text that stated "expected 3" or something like that

Reply:
oh yeah! I'll fix that!